updating/updating-restricted-network-cluster/index.adoc

```diff
@@ -13,6 +13,11 @@ If the local container registry and the cluster are connected to the mirror regi
 
 A single container image registry is sufficient to host mirrored images for several clusters in the disconnected network.
 
+== Mirroring the OpenShift Container Platform image repository
```

Comment:
This needs an ID above it, and you should use {product-title} here

Reply:
Good catch, thanks! I forgot to include the ID in this specific PR. Implementing both suggestions